<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1874</width>
    <height>842</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgba(158,177,175,255);</string>
  </property>
  <widget class="PlotWidget" name="Widget_1" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>911</width>
     <height>271</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
border-radius:20px;
</string>
   </property>
  </widget>
  <widget class="PlotWidget" name="Widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>10</y>
     <width>911</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
border-radius:20px;
</string>
   </property>
  </widget>
  <widget class="PlotWidget" name="Widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>560</y>
     <width>911</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
border-radius:20px;
</string>
   </property>
  </widget>
  <widget class="PlotWidget" name="Widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>911</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
border-radius:20px;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>300</y>
     <width>911</width>
     <height>531</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
border-radius:20px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>310</y>
     <width>611</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(158,177,175,255);
border-radius:20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="PushButton_UploadSignal">
   <property name="geometry">
    <rect>
     <x>1320</x>
     <y>320</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">  background: rgba(113,133,132,255);
  font-family: inherit;
  padding: 0.2em 0.3em;
  font-weight: 600;
  color:white;
  font-size: 14px;
  border: 3px solid rgba(68,82,83,255);
  border-radius: 0.4em;
  
</string>
   </property>
   <property name="text">
    <string>Upload</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>310</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Font-size:18px;
font-weight:1000;
color:white;
border-bottom:2px solid white;</string>
   </property>
   <property name="text">
    <string>Mixed Signals</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="SpinBox_Phase">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>450</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">  background: rgba(113,133,132,255);
  font-family: inherit;
  padding: 0.2em 0.3em;
  font-weight: 600;
  color:white;
  font-size: 14px;
  border: 3px solid rgba(68,82,83,255);
  border-radius: 0.4em;
  
</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="SpinBox_Amplitude">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>410</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">  background: rgba(113,133,132,255);
  font-family: inherit;
  padding: 0.2em 0.3em;
  font-weight: 600;
  color:white;
  font-size: 14px;
  border: 3px solid rgba(68,82,83,255);
  border-radius: 0.4em;
  
</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="SpinBox_Frequency">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>370</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">  background: rgba(113,133,132,255);
  font-family: inherit;
  padding: 0.2em 0.3em;
  font-weight: 600;
  color:white;
  font-size: 14px;
  border: 3px solid rgba(68,82,83,255);
  border-radius: 0.4em;
  
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>380</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Font-size:14px;
font-weight:1000;
color:white;</string>
   </property>
   <property name="text">
    <string>Frequancy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>420</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Font-size:14px;
font-weight:1000;
color:white;
</string>
   </property>
   <property name="text">
    <string>Amplitude</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>460</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Font-size:14px;
font-weight:1000;
color:white;
</string>
   </property>
   <property name="text">
    <string>Phase</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PushButton_AddComponent">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>540</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">  background: rgba(113,133,132,255);
  font-family: inherit;
  padding: 0.2em 0.3em;
  font-weight: 600;
  color:white;
  font-size: 14px;
  border: 3px solid rgba(68,82,83,255);
  border-radius: 0.4em;
  
</string>
   </property>
   <property name="text">
    <string>Add Component</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PushButton_GenerateSignal">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>540</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">  background: rgba(113,133,132,255);
  font-family: inherit;
  padding: 0.2em 0.3em;
  font-weight: 600;
  color:white;
  font-size: 14px;
  border: 3px solid rgba(68,82,83,255);
  border-radius: 0.4em;
  
</string>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>580</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Font-size:18px;
font-weight:1000;
color:white;
border-bottom:2px solid white;</string>
   </property>
   <property name="text">
    <string>Signals</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>1350</x>
     <y>580</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Font-size:18px;
font-weight:1000;
color:white;
border-bottom:2px solid white;</string>
   </property>
   <property name="text">
    <string>Components</string>
   </property>
  </widget>
  <widget class="QListWidget" name="ListWidget_Signals">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>630</y>
     <width>261</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">  background: rgba(113,133,132,255);

  border: 3px solid rgba(68,82,83,255);
  border-radius: 20px;
  
</string>
   </property>
  </widget>
  <widget class="QListWidget" name="ListWidget_Components">
   <property name="geometry">
    <rect>
     <x>1340</x>
     <y>630</y>
     <width>261</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">  background: rgba(113,133,132,255);

  border: 3px solid rgba(68,82,83,255);
  border-radius: 20px;
  
</string>
   </property>
  </widget>
  <widget class="QListWidget" name="ListWidget_SignalGeneration">
   <property name="geometry">
    <rect>
     <x>1310</x>
     <y>370</y>
     <width>291</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">  background: rgba(113,133,132,255);

  border: 3px solid rgba(68,82,83,255);
  border-radius: 20px;
  
</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Combobox_ReconstructionMethod">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>490</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">  background: rgba(113,133,132,255);
  font-family: inherit;
  padding: 0.2em 0.3em;
  font-weight: 600;
  color:white;
  font-size: 14px;
  border: 3px solid rgba(68,82,83,255);
  border-radius: 0.4em;
  
</string>
   </property>
   <item>
    <property name="text">
     <string>Shannon</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cubic_Splin</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nearest_Neighbor</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Linear</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>500</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Font-size:14px;
font-weight:1000;
color:white;
</string>
   </property>
   <property name="text">
    <string>Reconstrution</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1630</x>
     <y>310</y>
     <width>231</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(158,177,175,255);
border-radius:20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="Label_SetSamplingFrequancy">
   <property name="geometry">
    <rect>
     <x>1770</x>
     <y>510</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>HZ</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Checkbox_IsNormalizedSampling">
   <property name="geometry">
    <rect>
     <x>1660</x>
     <y>480</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Normal Range</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>1650</x>
     <y>570</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Font-size:18px;
font-weight:1000;
color:white;
border-bottom:2px solid white;</string>
   </property>
   <property name="text">
    <string>Default Signals</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>1660</x>
     <y>340</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Font-size:18px;
font-weight:1000;
color:white;
border-bottom:2px solid white;</string>
   </property>
   <property name="text">
    <string>Noise</string>
   </property>
  </widget>
  <widget class="QSlider" name="HorizontalSlider_SNR">
   <property name="geometry">
    <rect>
     <x>1670</x>
     <y>410</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(158,177,175,255);</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="Combobox_DefualtSignals">
   <property name="geometry">
    <rect>
     <x>1660</x>
     <y>610</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">  background: rgba(113,133,132,255);
  font-family: inherit;
  padding: 0.2em 0.3em;
  font-weight: 600;
  color:white;
  font-size: 14px;
  border: 3px solid rgba(68,82,83,255);
  border-radius: 0.4em;
  
</string>
   </property>
   <item>
    <property name="text">
     <string>All_Signals</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Signal 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Signal 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Signal 3</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="PushButton_SaveSignal">
   <property name="geometry">
    <rect>
     <x>1660</x>
     <y>750</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">  background: rgba(113,133,132,255);
  font-family: inherit;
  padding: 0.2em 0.3em;
  font-weight: 600;
  color:white;
  font-size: 14px;
  border: 3px solid rgba(68,82,83,255);
  border-radius: 0.4em;
  
</string>
   </property>
   <property name="text">
    <string>Save File</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>1790</x>
     <y>380</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>SNR</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEdit_SetSamplingFrequancy">
   <property name="geometry">
    <rect>
     <x>1660</x>
     <y>510</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QSlider" name="HorizontalSlider_SamplingFrequancy">
   <property name="geometry">
    <rect>
     <x>1660</x>
     <y>540</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(158,177,175,255);
  
</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="pageStep">
    <number>5</number>
   </property>
   <property name="tracking">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::NoTicks</enum>
   </property>
   <property name="tickInterval">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>1650</x>
     <y>440</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Font-size:18px;
font-weight:1000;
color:white;
border-bottom:2px solid white;</string>
   </property>
   <property name="text">
    <string>Sampling Rate</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEdit_SetSNRValue">
   <property name="geometry">
    <rect>
     <x>1660</x>
     <y>380</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <zorder>Widget_1</zorder>
  <zorder>Widget_4</zorder>
  <zorder>Widget_3</zorder>
  <zorder>Widget_2</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>PushButton_UploadSignal</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>label_11</zorder>
  <zorder>label_12</zorder>
  <zorder>ListWidget_Signals</zorder>
  <zorder>ListWidget_Components</zorder>
  <zorder>ListWidget_SignalGeneration</zorder>
  <zorder>SpinBox_Frequency</zorder>
  <zorder>SpinBox_Amplitude</zorder>
  <zorder>SpinBox_Phase</zorder>
  <zorder>label_14</zorder>
  <zorder>Combobox_ReconstructionMethod</zorder>
  <zorder>PushButton_GenerateSignal</zorder>
  <zorder>PushButton_AddComponent</zorder>
  <zorder>label_3</zorder>
  <zorder>Label_SetSamplingFrequancy</zorder>
  <zorder>Checkbox_IsNormalizedSampling</zorder>
  <zorder>label_16</zorder>
  <zorder>label_13</zorder>
  <zorder>HorizontalSlider_SNR</zorder>
  <zorder>Combobox_DefualtSignals</zorder>
  <zorder>PushButton_SaveSignal</zorder>
  <zorder>label_9</zorder>
  <zorder>LineEdit_SetSamplingFrequancy</zorder>
  <zorder>HorizontalSlider_SamplingFrequancy</zorder>
  <zorder>label_17</zorder>
  <zorder>LineEdit_SetSNRValue</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
